doc/rados/operations/cache-tiering.rst

```diff
@@ -1,6 +1,9 @@
 ===============
  Cache Tiering
 ===============
+.. warning:: Cache tiering has been deprecated in the Reef release as it
+             lacks a maintainer for a very long time. This does not mean
+             it will be certainly removed but the right for that is claimed.
```

Comment:
, but we may choose to remove it without much further notice.

Reply:
Changed.